File: myblog-server/src/main/java/com/site/blog/model/entity/Img.java
package com.site.blog.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.extension.activerecord.Model;
import lombok.*;

import java.io.Serial;
import java.io.Serializable;
import java.sql.Timestamp;
import java.time.LocalDateTime;

/**
 * @author yanni
 * @date time 2021/10/29 12:42
 * @modified By:
 */
@EqualsAndHashCode(callSuper = true)
@Data
@ToString
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Img extends Model<Img>  implements Serializable {
    @Serial
    private static final long serialVersionUID=1L;
    @TableId(  type = IdType.ASSIGN_ID)
    private String id;
    private String imgName;
    /**
     * 图片系统路径
     */
    private String imgPath;
    private Long imgSize;
    private String imgUrl;
    private String thumbnailPath;
    private String imgType;
    private String mediaType;
    private String md5;
    private Timestamp uploadTime;
}

File: myblog-kt/src/main/kotlin/com/site/blog/model/entity/Img.kt
package com.site.blog.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.extension.activerecord.Model
import com.fasterxml.jackson.annotation.JsonFormat
import java.io.Serial
import java.io.Serializable
import java.time.LocalDateTime

/**
 * @author yanni
 * @date time 2021/10/29 12:42
 * @modified By:
 */
data class Img(
    @TableId(type = IdType.ASSIGN_ID)
    var id: String? = null,
    var imgName: String? = null,

    /**
     * 图片系统路径
     */
    var imgPath: String? = null,
    var imgSize: Long? = null,
    var imgUrl: String? = null,
    var thumbnailPath: String? = null,
    var imgType: String? = null,
    var mediaType: String? = null,
    var md5: String? = null,

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    var uploadTime: LocalDateTime? = null,
) : Model<Img?>()